### PR TITLE
update with github personal access token and username

### PR DESCRIPTION
**Pull request by wjordan213**

### CODE REVIEW

File: .github/workflows/build-container.yml [hunk 1]
```diff
@@ -54,14 +61,16 @@ jobs:
...
+
           ECR_PATH='${{ inputs.ecr_path }}'
           ECR_PATH="${ECR_PATH:-"${{ inputs.path }}"}"
 
           DOCKER_BUILDKIT=1 docker build \
               --progress plain \
               --cache-from='${{ inputs.registry }}/${{ inputs.path }}:latest' \
               --build-arg BUILDKIT_INLINE_CACHE=1 \
+              --build-arg github_token="${{secrets.GH_TOKEN}}" \
+              --build-arg github_username="${{inputs.github_username}}" \
```
Comment (omgitsbillryan) [inline .github/workflows/build-container.yml:73]: Optionally adding these to the command only when they are defined could be a nice touch. 

As currently written, a Dockerfile that expects these args to be present will happily march along with the default `replace me` values, as opposed to quickly failing out complaining about missing args.

**Review by omgitsbillryan — APPROVED** [review]
LGTM. Had one small comment that I leave to your discretion.

File: .github/workflows/build-container.yml (hunk 1 above)
Comment (wjordan213) [reply .github/workflows/build-container.yml:73]: yeah, just trying to figure out the least painful way to do that in bash

**Comment by wjordan213** [comment]
I timeboxed conditionally adding the build args if they're defined. from what I can tell it's fairly complicated so I'm gonna merge and we can deal with this when this starts getting more use

**Pull request merged**